data-es6.js

```diff
@@ -6,6 +6,24 @@ var temp = {};
 var flag = "flagged";
 var fallthrough = "needs-polyfill-or-native";
 
+var typescript = {
+    compiler: { 
+        val: true, 
+        note_id: "typescript-compiler-or-native", 
+        note_html: "This feature is supported through a compiler error, or when the <code>--target ES6</code> flag is set and a native ES6 host is used." 
```

Comment:
Actually, let me get this clear: TS itself supports this error *unless* `--target ES6` is set, in which case it just falls through to the native platform? Hmm. I'd be inclined to change the wording to state it exactly as I have.